app/commands/help.js

```diff
@@ -0,0 +1,70 @@
+module.exports = Base => class extends Base {
+    constructor() {
+        super();
+
+        /**
+         * Expected to have the key be the command name.
+         * Then consist of an array that will work for the tellPlayerRaw method.
+         */
+        this.helpShortDescription = {};
+
+        // Full description is controlled fully by other files
+        this.helpFullDescription = {};
+    }
+
+    welcomeMessage(playerName) {
```

Comment:
Maybe consider having a more detailed welcome message for the first time you log in and a short "Welcome -Player-. Type !help for a list of commands." for any time you log in after.

Reply:
I can agree with having a shortened welcome message. And in the future this will be easy but currently there isn't an easy way to tell if this is the persons first login

Reply:
I can agree with having a shortened welcome message. And in the future this will be easy but currently there isn't an easy way to tell if this is the persons first login